PencilCase/PencilCase/scripts/application.ts

```diff
@@ -35,6 +36,17 @@ export class Application {
         this.activePage(homePage);
         this.homePage = ko.observable(homePage);
     }
+
+    public openDataBase = (): Database => {
```

Comment:
Add this method to use one db instance through the application, open db one time.